samples/dotnet/kernel-syntax-examples/Example31_CustomPlanner.cs

```diff
@@ -0,0 +1,364 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.XPath;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.CoreSkills;
+using Microsoft.SemanticKernel.Memory;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Planning;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Microsoft.SemanticKernel.Skills.Web;
+using Microsoft.SemanticKernel.Skills.Web.Bing;
+using RepoUtils;
+
+// ReSharper disable CommentTypo
+// ReSharper disable once InconsistentNaming
+internal static class Example31_CustomPlanner
```

Comment:
I think this example is awesome, but became too complex to fully understand.

I would split it into:
1. Creating a Custom Planner from a Custom Markup and Invoking it.
2. (Add new sample) Running Plans within Skills (Plan calling other Plans) 
3. Mixing the two above (Current example) with some reutilization.